src/AudacityApp.cpp

```diff
@@ -400,6 +406,74 @@ void PopulatePreferences()
    gPrefs->Flush();
 }
 
+wxDialog* MakePluginsCheckResultsDialog(const std::vector<wxString>& failed)
+{
+   auto dialog = safenew wxDialogWrapper(nullptr, wxID_ANY, XO("New Plugins"));
+   dialog->SetSize(635, 414);
+   auto layout = std::make_unique<wxBoxSizer>(wxVERTICAL);
```

Comment:
You do a lot with explicit sizers and control creations and so on.  Are you not comfortable enough with ShuttleGui?  All right if not, but a rewrite of this should be in the future to make all of this desirable plug-in checking platform neutral too.

Besides which:  perhaps this dialog belongs in its own small source file.

Reply:
>Are you not comfortable enough with ShuttleGui?

I'm still not very comfortable with wxWidgets)

>Besides which: perhaps this dialog belongs in its own small source file.

Probably, you are right